ambari-common/src/main/python/resource_management/libraries/functions/solr_cloud_util.py

```diff
@@ -117,6 +120,10 @@ def create_collection(zookeeper_quorum, solr_znode, collection, config_set, java
   appendableDict["--key-store-location"] = key_store_location
   appendableDict["--key-store-password"] = None if key_store_password is None else '{key_store_password_param!p}'
   appendableDict["--key-store-type"] = key_store_type
+  if default('configurations/infra-solr-env/infra_solr_ssl_enabled', False):
```

Comment:
it would only work if solr server or solr truststore is on that host where solr_cloud_util.py runs. the idea with that tool is that you can use from hosts where atlas or ranger is located.
therefore it would be better if ranger / atlas would pass their truststore data through the create_collections method. (so its expected for the ranger / atlas truststore should contain solr certs)

for atlas:
https://github.com/apache/ambari/blob/trunk/ambari-server/src/main/resources/common-services/ATLAS/0.1.0.2.3/package/scripts/metadata.py#L241

for ranger:
https://github.com/apache/ambari/blob/trunk/ambari-server/src/main/resources/common-services/RANGER/0.4.0/package/scripts/setup_ranger_xml.py#L747